libsqlite3-sys/build.rs

```diff
@@ -506,6 +507,7 @@ mod bindings {
 
     static PREBUILT_BINDGENS: &[&str] = &["bindgen_3.14.0"];
 
+    #[allow(clippy::needless_pass_by_value)]
```

Comment:
note that this is actually a bug in Clippy - I just fixed it there, hopefully will be merged and become available in a few months:  https://github.com/rust-lang/rust-clippy/pull/13113